charts/posit-chronicle/templates/stateful-set.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: chronicle-server
+  namespace: {{ $.Release.Namespace }}
+spec:
+  serviceName: chronicle-server
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      {{- include "posit-chronicle.pod.selectorLabels" . | trim | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "posit-chronicle.pod.labels" . | trim |  nindent 8 }}
+        {{- include "posit-chronicle.pod.selectorLabels" . | trim |  nindent 8 }}
+      annotations:
+        {{- include "posit-chronicle.pod.annotations" . | trim |  nindent 8 }}
+    spec:
+      {{- with .Values.pod.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.pod.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.pod.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if .Values.serviceaccount.create }}
+      serviceAccountName: chronicle-sa
+      {{- end }}
+      containers:
+      - name: chronicle-server
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+        command:
+        - {{ .Values.pod.command }}
+        {{- if .Values.pod.args }}
+        args:
+        {{- toYaml .Values.pod.args | nindent 8 }}
+        {{- end }}
+        ports:
```

Comment:
Are we missing the profiling port?

Reply:
i did a double take on this too, technically we don't need to expose it when we'd be accessing pprof using `kubectl port-forward`